applications/MeshingApplication/custom_utilities/mmg_utility.h

```diff
@@ -2280,7 +2390,7 @@ class MmgUtility
     {
         double* step_data = pNode->SolutionStepData().Data(step);
         
-        // NOTE: This just works with tetrahedron (you are going to have poblems with anything else)
+        // NOTE: This just works with tetrahedron (you are going to have problems with anything else)
```

Comment:
The problem with the prisms is that it is needed to change the the BinBasedFastPointLocator*, I will do that in a future, but it is an important change that requires some extra work. 

*The template of BinBasedFastPointLocator is BinBasedFastPointLocator<TDim>, and it just computes triangles and tetrahedron